libs/search-widget/src/old-components/search-input/SearchInput.svelte

```diff
@@ -32,26 +32,18 @@
   let inputContainerElement: HTMLElement | undefined;
   let position: DOMRect | undefined;
   let showSuggestions = false;
+  let hasFilters = false;
   const filters = nucliaState().filters.pipe(
-    tap((filterQuery) => {
-      if (popupSearch) {
-        typeAhead.set(filterQuery.join(''));
+    tap((filters) => {
+      // search box size changes when there are filters or not
+      if (hasFilters !== filters.length > 0) {
```

Comment:
that's a bit difficult to understand at first read, what about being more explicit:
```ts
const hasFiltersNow = filters.length > 0;
if (hasFilters !== hasFiltersNow) {
    hasFilters = hasFiltersNow;
…
```